config/configuration.json

```diff
@@ -964,5 +969,20 @@
       "difficulty":"NORMAL",
       "defaultPriority":"MEDIUM"
     }
+  },
+  "OceanBleedingCheck": {
```

Comment:
Could you also add the md doc for this check and update the available_checks.md file to include this check?